tools/devtool

```diff
@@ -556,6 +556,9 @@ cmd_test() {
 
     # If we got to here, we've got all we need to continue.
     say "Kernel version: $(uname -r)"
+    if [ "$(uname --machine)" = "x86_64" ]; then
+        say "Microcode version: $(sudo cat /sys/devices/system/cpu/cpu0/microcode/version)"
```

Comment:
Can we drop the `sudo` here (or more it into the "CI only" block below)? Its causing some inconvenience when running outside of our CI :(

Reply:
I didn't want to add the `sudo` there too. I saw an error of lack of access right somewhere, but I cannot find it now... Let me try to remove it in PR #4118

Reply:
https://buildkite.com/firecracker/firecracker-pr-optional/builds/2649#018ab343-b2c0-4ee9-b239-f665b4fc9531
```
cat: /sys/devices/system/cpu/cpu0/microcode/version: Permission denied
--
  | [Firecracker devtool 2023-09-20T15:50:11+0000] Microcode version:
```

Reply:
It appears to happen only on 4.14 kernel.

Reply:
Can we just move it to the "if [[ $BUILDKITE == "true ]];" block a few lines down? I don't think its very important to have this output locally (which is where it prompts me for a password now)

Reply:
Yes, we can, but I'd like to show it soon below the kernel version :(
Let me think another approach.

Reply:
/proc/cpuinfo solves this problem.